src/videojs-contrib-hls.js

```diff
@@ -273,6 +273,8 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      videojs.plugin('gapSkipper', gapSkipper);
```

Comment:
You should be able to register this at the bottom where we do videojs.registerComponent